Editor/Utility/VcamStageEditor.cs

```diff
@@ -341,6 +354,18 @@ public void OnInspectorGUI(bool withHeader)
             }
         }
 
+        public void OnSceneGUI()
+        {
+            for (int i = 0; i < m_subeditors.Length; ++i)
+            {
+                var ed = m_subeditors[i];
+                if (ed == null)
+                    continue;
+                
+                ed.OnSceneGUI();
+            }
+        }
+
         // Pass the dragged event down to the CM component editors
         public void OnPositionDragged(Vector3 delta)
```

Comment:
This mechanism is no longer needed.  Can be removed if not an API-breaking change